client/app/scripts/charts/node.js

```diff
@@ -55,29 +54,20 @@ class Node extends React.Component {
     this.saveShapeRef = this.saveShapeRef.bind(this);
   }
 
-  componentWillReceiveProps(nextProps) {
-    // marks as matched only when search query changes
-    if (nextProps.searchQuery !== this.props.searchQuery) {
-      this.setState({ matched: nextProps.matched });
-    } else {
-      this.setState({ matched: false });
-    }
-  }
```

Comment:
**Note to myself**: Put back some of this code as the `throb` animation doesn't get triggered now.